go2ts_test.go

```diff
@@ -80,21 +81,26 @@ export interface ComplexStruct {
 	fe?: number;
 	te?: string;
 	oe?: OtherStruct | null;
-	d: { [key: string]: any };
-	dp: { [key: string]: any } | null;
+	d: Data;
+	dp: Data | null;
 	mss: { [key: string]: string[] };
 	mik: { [key: number]: string };
+	mos: { [key: string]: OtherStruct };
 	slice: string[] | null;
 	sos: string[][] | null;
-	sod: { [key: string]: any }[] | null;
-	mod: { [key: string]: { [key: string]: any } };
-	mosod: { [key: string]: { [key: string]: any }[] };
-	momosod: { [key: string]: { [key: string]: { [key: string]: any }[] } };
-	mode: string;
+	sod: Data[] | null;
+	mod: { [key: string]: Data };
+	mosod: { [key: string]: Data[] };
+	momosod: { [key: string]: { [key: string]: Data[] } };
+	mode: Mode;
 	inline: Anonymous1;
 	array: string[];
 	Color: Alpha;
 }
+
+export type Data = { [key: string]: any };
+
+export type Mode = string;
```

Comment:
Can we have a type alias for a primitive type other than string?

Reply:
Yes, added a number example.